en/micro-integrator/docs/references/connectors/salesforce-rest-connector/sf-rest-connector-example.md

```diff
@@ -21,16 +26,123 @@ Follow these steps to set up the ESB Solution Project and the Connector Exporter
+
+    For retrieve data from the created objects in the salesforce account,you need to add the `query` operation to the `retrieve` sequence. 
+    
+    - **queryString** :  This variable contains specified SOQL query. In this sample this SOQL query executes to retrive `id` and `name` from created `Account`. If the query results are too large, the response contains the first batch of results.
+                                                                                                                                                                     
+    Navigate into the **Palette** pane and select the graphical operations icons listed under **Salesforce Connector** section. Then drag and drop the `query` operations into the Design pane.      
+    
+    <img src="../../../../assets/img/connectors/salesforce-drag-and-drop-query.png" title="Add query operation to retrive sequnce" width="800" alt="Add query operation to retrive sequnce"/> 
+    
+    Select the query operation and add `id, name from Account` query to the properties section shown as bellow.
+    
+    <img src="../../../../assets/img/connectors/salesforce-api-retrive-query-operation-sequnce.png" title="Add query to the query operation in retrive sequnce" width="800" alt="Add query to the query operation in retrive sequnce"/>
+    
+4. Configure the `salesforcerest API` using the created `create` and `retrive` sequences.
+ 
+    Now you can select the API which we created in step 1. Navigate into the **Palette** pane and select the graphical operations icons listed under **Defined Sequences** section. Then drag and drop the created `create` and `retrive` sequences to the Design pane.
+    
+    <img src="../../../../assets/img/connectors/salesforce-drag-and-drop-sequencestothe-DesignPane.png" title="Drag and drop sequences to the Design view" width="800" alt="Drag and drop sequences to the Design view"/> 
+ 
+5. Getting response to the user
+    
+    When you invoking the created API the request of the message is going through the `create` and `retrive` sequences. Finally pass to the the [Respond mediator](https://ei.docs.wso2.com/en/next/micro-integrator/references/mediators/respond-Mediator/). In here the Respond Mediator stops the processing on the current message and sends the message back to the client as a response.            
+    
+    Drag and drop **respond mediator** to the **Design view**. 
+    
+    <img src="../../../../assets/img/connectors/salesforce-drag-and-drop-respond-mediator.png" title="Add Respond mediator" width="800" alt="Add Respond mediator"/> 
+
+    Once you have setup the sequences and API, you can see the `salesforcerest` API as following manner.
+    
+    <img src="../../../../assets/img/connectors/salesforce-api-design-view.png" title="API Design view" width="800" alt="API Design view"/>
+       
+6.  Now you can switch into the Source view and check the XML configuration files of the created API and sequences. 
```

Comment:
IMO, this is not a step. We can just mention it without including it into a step.

Reply:
You can mention XML config is auto generated when we drag and drop mediators and connector operations to the design pallet.

Reply:
Updated.